.travis.yml

```diff
@@ -1,5 +1,3 @@
 language: ruby
 rvm:
- - 2.1.2
- - 2.1.8
- - 2.2.4
+ - 2.3.0
```

Comment:
Removing the other versions may make it harder to get these changes merged upstream